File: src/main/java/com/hedvig/claims/serviceIntegration/paymentService/PaymentService.java
package com.hedvig.claims.serviceIntegration.paymentService;

import com.hedvig.claims.serviceIntegration.paymentService.dto.PaymentResponse;
import com.hedvig.claims.serviceIntegration.paymentService.dto.PayoutRequest;

public interface PaymentService {

  PaymentResponse executePayment(String memberId, PayoutRequest request);

}

File: src/main/java/com/hedvig/claims/serviceIntegration/paymentService/PaymentService.kt
package com.hedvig.claims.serviceIntegration.paymentService

import com.hedvig.claims.serviceIntegration.paymentService.dto.PaymentResponse
import com.hedvig.claims.serviceIntegration.paymentService.dto.PayoutRequest

interface PaymentService {
    fun executePayment(memberId: String, request: PayoutRequest): PaymentResponse
}
